packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -30,7 +30,7 @@ import 'tooltip_theme.dart';
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=EeEfD5fI-5Q}
 ///
-/// {@tool dartpad --template=stateless_widget_material}
+/// {@tool dartpad --template=stateless_widget_scaffold_center}
```

Comment:
This means that the `Center` widget here and in the other sample can also be removed, since it'd be redundant :)

Reply:
ohh yes sorry for that
let me change again